test/minitest/test_minitest_test.rb

```diff
@@ -848,19 +848,25 @@ def non_verbose
     $VERBOSE = orig_verbose
   end
 
+  def sample_test_case(rand = nil)
+    srand(rand) if rand
+    Class.new FakeNamedTest do
+      100.times do |i|
+        define_method("test_#{i}".to_sym) { assert true }
```

Comment:
you never need to `to_sym` for `define_method`

Reply:
I'll try, but Ruby's doc seems to only show a symbol as a parameter...

Reply:
You're correct.  Fixed.